### PR TITLE
Update to rocky linux 9

### PR DESCRIPTION
**Pull request by khaledk2**
This PR updates  the role to use Rocky Linux 9

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -4,11 +4,11 @@ galaxy_info:
   description: Create or remove local user accounts
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.1
+  min_ansible_version: 2.10
```
Comment (pwalczysko) [inline meta/main.yml:7]: Maybe then leave this at 2.1 ? As discussed this morning @jburel - I cannot see any dependencies here.

Comment (jburel) [reply meta/main.yml:7]: I will keep it as it was

Comment (jburel) [reply meta/main.yml:7]: 2.1 does not receive security update. so this should probably be bumped
Following this am discussion I checked the timeline
2.x including 2.10 have reached end of Life

Comment (pwalczysko) [reply meta/main.yml:7]: > 2.1 does not receive security update. so this should probably be bumped Following this am discussion I checked the timeline 2.x including 2.10 have reached end of Life

There might be a confusion between ansible and ansible-core versions. Unfortunately, when ``ansible --version`` is run, only the ansible-core version is nicely displayed. We are installing ansible in our tests with ``pip install ansible<8`` which installs ansible 7.7.0 (see below)

```
(ansible-supernew) pwalczysko@Petrs-MBP ~ % python --version
Python 3.10.13
(ansible-supernew) pwalczysko@Petrs-MBP ~ % 
(ansible-supernew) pwalczysko@Petrs-MBP ~ % 
(ansible-supernew) pwalczysko@Petrs-MBP ~ % pip install "ansible<8"
Collecting ansible<8
  Downloading ansible-7.7.0-py3-none-any.whl.metadata (7.9 kB)
Collecting ansible-core~=2.14.7 (from ansible<8)
  Downloading ansible_core-2.14.13-py3-none-any.whl.metadata (6.9 kB)
```

So yes, the message in the README is totally flawed and outdated.

Comment (pwalczysko) [reply meta/main.yml:7]: > So yes, the message in the README is totally flawed and outdated.

Although maybe not, it depends on what you call "ansible version". To see the ``ansible version`` which is NOT the ansible-core version, use ``ansible-community``

```
(ansible-supernew) pwalczysko@Petrs-MBP ~ % ansible-community --version
Ansible community version 7.7.0
```

Comment (pwalczysko) [reply meta/main.yml:7]: Maybe we should specify if we mean the ``ansible-community`` or the ``ansible-core`` versions

https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html

Comment (jburel) [reply meta/main.yml:7]: Something we probably need to be clear about

Comment (pwalczysko) [reply meta/main.yml:7]: @jburel @khaledk2 : I think line we are discussing about is a galaxy variable.
According to https://github.com/ansible/galaxy/issues/3114#issuecomment-1643960145 this variable is pointing to ``ansible-core``.

So, we might just add a comment above that variable, something like ``#points to the version of the ansible-core package`` above this line ?

Comment (jburel) [reply meta/main.yml:7]: that will make sense

Comment (pwalczysko) [reply meta/main.yml:7]: @jburel Nevertheless, your original worry is completely valid. I think ``ansible-core`` 2.1 is outdated too. https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix - I would actually go for 2.13 everywhere (even that is EOL, but the speed of development there is breathtaking)

**Review by jburel — APPROVED** [review]

**Pull request merged**